docs/content/en/docs/migrate/metrics-observe/_index.md

```diff
@@ -2,51 +2,335 @@
   using Helm charts and standard practices.
+* Keptn supports multiple instances of multiple data providers.
 * You must populate a
   [KeptnMetricsProvider](../../yaml-crd-ref/metricsprovider.md) resource
   for each instance of each data provider.
   This resource specifies the URL and namespace for the data provider
-  and gives it a unique `name` that can be referenced by other resources.
-  This is the only Keptn specific configuration that is required.
-* Each query that is specified in your Keptn v1
-  [slo.yaml](https://keptn.sh/docs/1.0.x/reference/files/sli/) file
-  should be converted into its own
-  [KeptnMetric](../../yaml-crd-ref/metric.md) resource
-  if you are using it for an evaluation.
-  Note that Keptn supports using multiple instances of multiple data providers.
-* Simple Keptn v1 comparisons that are defined in
+  and gives it a unique `name` that can be referenced
+  for Keptn Metrics, Evaluations, and Analysis.
+* Queries and objectives that are specified in your Keptn v1
+  [sli.yaml](https://keptn.sh/docs/1.0.x/reference/files/sli/)
+  and
+  [slo.yaml](https://keptn.sh/docs/1.0.x/reference/files/sli/)
+  files are transferred/converted into Keptn resources.
+
+## Transfer Keptn v1 SLIs/SLOs to evaluation resources
+
+Simple comparisons of data can be implemented as
+[Keptn Evaluations](../../implementing/evaluations.md).
+To implement this:
+
+* Transfer the information from the Keptn v1
+  [sli.yaml](https://keptn.sh/docs/1.0.x/reference/files/sli/)
+  files into
+  [KeptnMetric](../../yaml-crd-ref/metric.md) resources
+* Transfer the information from the Keptn v1
   [slo.yaml](https://keptn.sh/docs/1.0.x/reference/files/slo/)
-  files can be converted to
+  files into
   [KeptnEvaluationDefinition](../../yaml-crd-ref/evaluationdefinition.md)
   resources.
-* Keptn v1 calculations that use weighting and scoring
-  cannot currently be implemented for Keptn
-  but this functionality is under development.
 
-For more information about working with Keptn metrics, see the
-[Keptn Metrics](../../implementing/evaluatemetrics.md)
-page.
+## Convert Keptn v1 SLIs/SLOs to Analysis resources
+
+The Keptn Analysis feature provides capabilities
+similar to those of the Keptn v1
+[Quality Gates](https://keptn.sh/docs/1.0.x/define/quality-gates/)
+feature
+but it uses Kubernetes resources to define the analysis to be done
+rather than the configuration files used for Keptn v1.
+Tools are provided to convert Keptn v1 SLI and SLO definitions
+to Keptn Analysis resources.
+
+These instructions assume that the same SLO file
+has been used for all services in the project,
+so only one `AnalysisDefinition` resource
+(named `my-project-ad`) is created.
+If your Keptn v1 project has multiple SLOs,
+you need to create a separate `AnalysisDefinition`
+with a unique namefor each one.
+
+The process is:
+
+1. If necessary, rename the SLI to a Kubernetes compatible name
+
+   Keptn requires that all SLI names adhere to the
```

Comment:
I just did a check regarding this, and in the meantime the conversion tool automatically takes care of the renaming, so the user does not need to do this on their own. We should keep this as a heads up though, to make users of the converter aware that the names of their SLIs might end up having different names

Reply:
Wonderful news!  So I deleted the step to manually rename the SLIs and instead added a note to step for converting the SLIs that explains how and why the names are converted.